src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -48,6 +48,10 @@
   <module name="org.openhab.tools.analysis.checkstyle.OverridingParentPomConfigurationCheck">
     <property name="severity" value="warning" />
   </module>
+  
+  <module name="org.openhab.tools.analysis.checkstyle.RequireBundleCheck">
+      <property name="severity" value="warning" />
```

Comment:
I didn't yet check them in detail, but if you have the impression that Require-Bundle isn't required in them: Yes, please, create a PR!